lib/pg_easy_replicate/query.rb

```diff
@@ -10,20 +10,22 @@ def run(
         connection_url:,
         user: internal_user_name,
         schema: nil,
-        transaction: true
+        transaction: true,
+        timeout: nil
       )
         conn =
           connect(connection_url: connection_url, schema: schema, user: user)
+        timeout ||= "5s"
```

Comment:
I'd say lets drop the timeout being introduced in this PR for now. I have an idea to just reset the statement time out before vaccum/analyze and then restore it. This way, its fine if the vacuum is going over